graphs/possible_bipartition.py

```diff
@@ -5,8 +5,48 @@ def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
```

Comment:
⏱ Time complexity would actually be O(N + E) where N is the number of nodes (aka dogs) in the graph and E is the number of edges (dislike between two dogs) in the graph represented by `dislikes`. This is because you visit each edge and each node exactly once.